dev_guide/pod_autoscaling.adoc

```diff
@@ -191,6 +196,22 @@ apiServerArguments:
 ...
 ----
 
+.. Restart the {product-title} services:
++
+----
+# master-restart api
```

Comment:
This uses # as a prompt where other steps use $, is that correct, or should they all be the same?